examples/game_gomoku/main.ipynb

```diff
@@ -82,13 +91,13 @@
     "        \n",
     "    for y in range(size):\n",
     "        for x in range(size):\n",
-    "            if board[y, x] == NAME_TO_PIECE[NAME_BLACK]:  # black player\n",
+    "            if board[y, x] == NAME_TO_PIECE[NAME_WHITE]:  # white player\n",
```

Comment:
Why change to white player here? Lines 95-98 is plotting a black piece, and lines101-103 is plotting a white piece.

Reply:
Meanwhile, after this change, there is inconsistence and error in the game as follows
![image](https://github.com/modelscope/agentscope/assets/102287034/dc1930af-6db8-436e-b19f-a5818d4553cb)